Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Ride.hs

```diff
@@ -204,3 +210,10 @@ currentActiveRide :: ShortId DM.Merchant -> ApiTokenInfo -> Text -> FlowHandler
 currentActiveRide merchantShortId apiTokenInfo vehichleNumber = withFlowHandlerAPI $ do
   checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
   Client.callDriverOfferBPP checkedMerchantId (.rides.currentActiveRide) vehichleNumber
+
+ticketRideList :: ShortId DM.Merchant -> ApiTokenInfo -> Maybe (ShortId Common.Ride) -> Maybe Text -> Maybe Text -> Maybe Text -> FlowHandler Common.TicketRideListRes
+ticketRideList merchantShortId apiTokenInfo mbRideShortId mbCountryCode mbPhoneNumber mbSupportPhoneNumber = withFlowHandlerAPI $ do
+  checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
+  transaction <- buildTransaction Common.TicketRideListEndpoint apiTokenInfo Nothing T.emptyRequest
```

Comment:
Not needed as it is a get request

Reply:
https://github.com/nammayatri/nammayatri/pull/1759#discussion_r1280180294

Reply:
since it is for third party so we are storing it